### ISSUE TITLE
Add A Bottom Toolbar

### ISSUE THREAD

**Issue by gitdebpro** [bromite/bromite#1262]
Hi there, I know that the main concept of bromite is privacy and not altering the main chromium interface but nowadays phone screens are huge its really impossible to tap on the top search bar for searching anything. So I am requesting you to reconsider this only :

1. An option to enable a bottom bar.

2. The Bottom Bar will consist of Home, Search, Tab Switcher and Menu Button only.

3. The Bottom Bar will autohide like the top bar on Scrolling.

In a Nutshell I mean to say a Bottom Bar like Brave Browser has implemented for Android.

Thanks.

**Comment by kokukyocho** [comment]
Hello @gitdebpro!

I am a bot :robot:

Thanks for submitting this issue!
I noticed that it is missing the template, please edit the issue to match the template for either a [bug](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) or a [feature request](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/feature_request.md).

The issue will be automatically re-opened after that.

**Tip**: if you use GitHub in Desktop mode the template will be proposed to you when submitting issues.
